index.html

```diff
@@ -34,15 +34,7 @@ <h1>I’m Tshepo. <br> Glad to see <br> you!</h1>
   
 </div>
 
-
-
-
-
-
-   
-
-
-    <div class="Container2">
+   <div class="Container2">
```

Comment:
- [ ] Following the best practices in using the correct semantic tags, the Work me section should be inside a `section tag` and not a `div`, I'll suggest you update this :+1: :waffle: